tests/integration/test_quorum_inserts/test.py

```diff
@@ -46,10 +47,11 @@ def started_cluster():
 
 
 def test_simple_add_replica(started_cluster):
-    zero.query("DROP TABLE IF EXISTS test_simple ON CLUSTER cluster")
+    table_name = "test_simple_" + uuid.uuid4().hex
+    zero.query(f"DROP TABLE IF EXISTS {table_name} ON CLUSTER cluster")
```

Comment:
I think this `DROP TABLE` statements are not the best. Because of the uuid, the table probably doesn't exists, but if it exists, then we should use `SYNC` to make sure the is actually dropped before we try to create it. Usually dropping is fast, but it might be slow on heavy buidls.

I would suggest just to remove them completely and take the risk of colliding uuids